it/controllers/AppealsControllerISpec.scala

```diff
@@ -382,30 +382,6 @@ class AppealsControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock
       |}
       |""".stripMargin)
 
-  "getAppealsDataForLateSubmissionPenalty" should {
-
-    "when the 1812 feature switch is enabled" must {
```

Comment:
I didn't mean to remove the whole test just the 'when the 1812 feature switch is enabled' part so it should look like:
"getAppealsDataForLateSubmissionPenalty" must {
"call ETMP and compare the penalty ID provided and the penalty ID in the payload - return OK if there is a match" in {
etc.